packages/lib-classifier/src/helpers/validateMimeType/validateMimeType.spec.js

```diff
@@ -0,0 +1,46 @@
+import validateMimeType from './validateMimeType'
+
+const validMimeTypes = [
+  'application/json',
+  'text/plain',
+  'image/png',
+  'image/jpeg',
+  'image/gif',
+  'audio/mp3',
+  'audio/m4a',
+  'audio/mpeg',
+  'video/mp4',
+  'video/mpeg',
+  'video/x-m4v'
+]
+
+const invalidMimeTypes = [
+  'application/xml',
+  'audio/aac',
+  'audio/wave',
+  'image/tiff'
```

Comment:
Maybe add `text/html` to the list of blocked types?

Reply:
I’m being paranoid, but could you block JS here too?

Reply:
I've added a few more invalid mime types.